### ISSUE TITLE
ALL_TOPICS client subscription not working

### ISSUE THREAD

**Issue by jwa1** [authorizon/fastapi_websocket_pubsub#8]
When subscribing to ALL_TOPICS from a client they do not receive any updates.

server.py:
```from fastapi_websocket_pubsub.event_notifier import ALL_TOPICS
import uvicorn
from fastapi import FastAPI
from fastapi_websocket_pubsub import PubSubEndpoint
import asyncio
import uvicorn
from fastapi import FastAPI

app = FastAPI()
endpoint = PubSubEndpoint()
endpoint.register_route(app, "/pubsub")


async def on_events(subscription, data):
    print(f"Hello from {subscription.topic}: {data}")


@app.on_event("startup")
async def startup_client():
    # Subscribe to all incoming messages
    await endpoint.subscribe(ALL_TOPICS, on_events)


async def events():
    # await asyncio.sleep(1)
    await endpoint.publish("other", data={"info": 3, "some_data": "test123"})
    await asyncio.sleep(1)
    await endpoint.publish("other", data={"info": 2, "some_data": "test456"})
    await asyncio.sleep(1)
    await endpoint.publish("test", data={"info": 1, "some_data": "test999"})


@app.get("/request")
async def trigger_events():
    asyncio.create_task(events())


if __name__ == "__main__":

    uvicorn.run(
        "server:app",
        host="0.0.0.0",
        reload=False,
        port=8020,
    )
```

client.py:
```from fastapi.applications import FastAPI

from fastapi_websocket_pubsub.event_notifier import ALL_TOPICS
import uvicorn

from fastapi_websocket_pubsub import PubSubClient

app = FastAPI()
endpoint = PubSubClient()


async def on_events(data, topic):
    print(f"Received from {topic}: {data}")


@app.on_event("startup")
async def startup_client():
    # Subscribe to all incoming messages
    endpoint.subscribe(ALL_TOPICS, on_events)
    endpoint.start_client("ws://0.0.0.0:8020/pubsub")


@app.get("/")
async def get():
    await endpoint.publish("test", data={"info": 1, "some_data": "client"})


if __name__ == "__main__":

    uvicorn.run(
        "client:app",
        host="0.0.0.0",
        reload=False,
        port=8010,
    )
```

Expected behavior:
client.py will receive all data from server.py
server.py receives all data

Observed behaviour:
client.py does not receive any data
server.py receives all data

**Comment by orweis** [comment]
Hi @jwa1 - just letting you know I've seen this; and will dive into it a little later :)

**Comment by orweis** [comment]
Indeed an issue :) @jwa1 Thanks for pointing this out
The ALL_TOPICS feature was planned just for the server-side (to support the broadcaster mechanism), but there's no reason why we shouldn't enable it for the client-side as well. 

I've added a fix, and tests - https://github.com/authorizon/fastapi_websocket_pubsub/pull/9
I've asked @asafc to do a quick CR - and once done we'll also publish a new version to PyPi.

**Comment by orweis** [comment]
Ok, was fast.
Released to PyPi as 0.1.15
https://github.com/authorizon/fastapi_websocket_pubsub/releases/tag/v0.1.15

Enjoy

**Comment by jwa1** [comment]
Thanks @orweis - appreciate the quick response + fix!
